src/OpenGL/Silk.NET.OpenGL.Abstactions/BufferObject.cs

```diff
@@ -0,0 +1,70 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Silk.NET.OpenGL.Abstactions
```

Comment:
Please delete this project, we don't want to provide our own abstractions package (yet)